dnf-testing.sh

```diff
@@ -9,4 +15,11 @@ if [ ! $PYTHON ]; then
     exit 1
 fi
 
-exec "$PYTHON" dnf-testing.py "$@"
+FEATURESDIR="features"
+if [[ $PROG_NAME == "createrepo_c-testing.sh" ]]; then
+    FEATURESDIR="createrepo_c"
+fi
+
+exec "$PYTHON" dnf-testing.py --featuresdir=$FEATURESDIR "$@"
```

Comment:
This is kind of complex and again, wrappers wrappers. Seems unnecessary to me, why not just have (using the current script name even though I suggested finding a better one):
```
# dnf-testing.py dnf ...
# dnf-testing.py createrepo_c ...
```